hazelcast/src/main/java/com/hazelcast/spi/impl/servicemanager/impl/ServiceManagerImpl.java

```diff
@@ -274,7 +274,7 @@ private void shutdownService(final ManagedService service, final boolean termina
         }
     }
 
-    private synchronized void registerService(String serviceName, Object service) {
+    protected synchronized void registerService(String serviceName, Object service) {
```

Comment:
If we provide a way to register services with a programmatic API, we might not need this change since this class is opened for extension just for service registration purposes (See https://github.com/hazelcast-incubator/cascading-prototype/blob/transport/hazelcast-jet/src/main/java/com/hazelcast/jet/impl/hazelcast/JetServiceManagerImpl.java).